flink-runtime/src/test/java/org/apache/flink/runtime/operators/hash/HashTableITCase.java

```diff
@@ -735,10 +728,9 @@ public void testSparseProbeSpilling() throws IOException, MemoryAllocationExcept
                 numRecordsInJoinResult++;
             }
         }
-        Assert.assertEquals(
-                "Wrong number of records in join result.",
-                expectedNumResults,
-                numRecordsInJoinResult);
+        assertThat(numRecordsInJoinResult)
+                .withFailMessage("Wrong number of records in join result.")
+                .isEqualTo(expectedNumResults);
```

Comment:
I cannot comment at `public void testSparseProbeSpillingWithOuterJoin()` line, so comment here.

the public can be removed.